build.sbt

```diff
@@ -1,8 +1,17 @@
-import sbt._
 import codacy.libs._
+import sbt._
 
 Universal / javaOptions ++= Seq("-Xms1g", "-Xmx2g", "-Xss512m", "-XX:+UseG1GC", "-XX:+UseStringDeduplication")
 
+val assemblyCommon = Seq(
+  test in assembly := {},
+  assemblyMergeStrategy in assembly := {
+    case "META-INF/versions/9/module-info.class" => MergeStrategy.discard
```

Comment:
Okey.... is this like copy pasta from somewhere, or that is happening here?

Reply:
So, basically, when running the command to generate the fat jar `sbt codacyAnalysisCli/assembly`, I was getting the following error:

```
[error] (codacyAnalysisCli / assembly) deduplicate: different file contents found in the following:
[error] /Users/jose/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/bouncycastle/bcpg-jdk15on/1.64/bcpg-jdk15on-1.64.jar:META-INF/versions/9/module-info.class
[error] /Users/jose/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/bouncycastle/bcpkix-jdk15on/1.64/bcpkix-jdk15on-1.64.jar:META-INF/versions/9/module-info.class
[error] /Users/jose/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/bouncycastle/bcprov-jdk15on/1.64/bcprov-jdk15on-1.64.jar:META-INF/versions/9/module-info.class
```

After a search for the error, I found this https://stackoverflow.com/questions/54834125/sbt-assembly-deduplicate-module-info-class with the current solution for the error.

Reply:
Interesting, maybe we can make a comment here just before the assemblyCommon? Something like Hack, or //Workaround: https://stackoverflow.com/questions/54834125/sbt-assembly-deduplicate-module-info-class what do you think?